lib/ansible/utils/module_docs_fragments/manageiq.py

```diff
@@ -0,0 +1,36 @@
+#
+# (c) 2017, Daniel Korn <korndaniel1@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+class ModuleDocFragment(object):
+
+    # Standard ManageIQ documentation fragment
+    DOCUMENTATION = """
+options:
+  miq:
+    required: true
+    description:
```

Comment:
I tried, and did not find how to implement this :-(

When I wrote it as a regular `options:` list, rendered using `ansible-doc ` it was on the same level as the `managiq_user` `options:` and when I tried to set a new level of `['options']['options']` in the yaml, I got syntex errors form `ansible-doc `

What am I missing ? can you give a small example, how to do this ?